projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/plugin_entry.py

```diff
@@ -23,6 +23,7 @@ def run_job(context: JobContext) -> None:
         meta_job_runner.META_CONFIG = MetaPluginConfiguration(
             context.core_context.configuration
         )
+        meta_job_runner.ARGUMENTS = context.job_args.get_arguments()
```

Comment:
What is this?

Reply:
Related to https://github.com/vmware/versatile-data-kit/pull/1839#discussion_r1157123874
This way we get the args, passed by --arguments flag.

Reply:
Removed it.